renovate.json

```diff
@@ -68,14 +68,16 @@
       "enabled": false
     },
     {
+      "description": "Packages which need major versions to be in sync with the EF Core version or the .NET version",
       "matchDatasources": ["nuget"],
       "matchPackagePrefixes": [
         "dotnet-ef",
         "Microsoft.AspNetCore",
         "Microsoft.EntityFrameworkCore",
         "Microsoft.Extensions",
         "Thinktecture.EntityFrameworkCore",
-        "Microsoft.VisualStudio.Web.CodeGeneration.Design"
+        "Microsoft.VisualStudio.Web.CodeGeneration.Design",
+        "MockQueryable.Moq"
```

Comment:
I added `MoqQueryable.Moq` to the list of packages ignored from major version updates here due its dependency on `Microsoft.EntityFrameworkCore` >= 7.0.0.